py/picca/pk1d/postproc_pk1d.py

```diff
@@ -737,10 +735,15 @@ def compute_average_pk_redshift(
                 standard_dev, _, _ = binned_statistic(
                     data_snr, p1d_values, statistic="std", bins=snr_bin_edges
                 )
+                standard_dev_error, _, _ = binned_statistic(
```

Comment:
this is for rebinning the statistics of interest? I.e. the individual P1d estimates for raw/noise/..., reso correction and so on?
And the line above is for the final P1d estimate (which is part of the P1d_table as well?
In both cases weigthning is done with the forest_snr column?

I guess in principle one could precompute p1d_values, standard_dev  (and everything else that is independent of the column of interest) outside the loop and not compute them once per column, except if you have pixels masked in a column (because of NaN values), but not in the p1d (which I'd consider weird as a NaN should have propagated to the final p1d)...

Reply:
also the naming convention is weird, shouldn't this be `standard_dev_col` and the other one `standard_dev_p1d` or similar?

Reply:
I agree on changing the naming convention and on precomputing the p1d, I will make the change.